src/cryptsetup/meson.build

```diff
@@ -32,3 +33,9 @@ executables += [
                 'sources' : files('cryptsetup-generator.c'),
         },
 ]
+
+if conf.get('HAVE_LIBCRYPTSETUP') == 1
```

Comment:
can we conditionalize under some COMPAT build time option please?

Reply:
We *could*, but we generally don't do this. The extra symlink is completely negligible and then we can drop it in a few releases. It's not worth creating another option for this.

Reply:
Added.